_includes/global-header.html

```diff
@@ -10,24 +10,18 @@
                     <div class="navbar-collapse collapse">
                         <ul class="primary nav navbar-nav">
                             <li><a href="https://docker.com/why-docker">Why Docker?</a></li>
-                            <li><a href="https://docker.com/get-started">Product</a></li>
+                            <li><a href="https://www.docker.com/products">Products</a></li>
```

Comment:
Looks like we use `www.` as the canonical URL, so we should probably do a find/replace across the repository

`https://docker.com/` -> `https://www.docker.com/`

Reply:
Sure, will fix that.